collector/receiver/telemetryapireceiver/receiver.go

```diff
@@ -216,16 +216,16 @@ func (r *telemetryAPIReceiver) createLogs(slice []telemetryapi.Event) (plog.Logs
 	scopeLog.Scope().SetName(scopeName)
 	for _, el := range slice {
 		r.logger.Debug(fmt.Sprintf("Event: %s", el.Type), zap.Any("event", el))
-		logRecord := scopeLog.LogRecords().AppendEmpty()
-		logRecord.Attributes().PutStr("type", el.Type)
-		if t, err := time.Parse(timeFormatLayout, el.Time); err == nil {
-			logRecord.SetTimestamp(pcommon.NewTimestampFromTime(t))
-			logRecord.SetObservedTimestamp(pcommon.NewTimestampFromTime(time.Now()))
-		} else {
-			r.logger.Error("error parsing time", zap.Error(err))
-			return plog.Logs{}, err
-		}
 		if el.Type == string(telemetryapi.Function) || el.Type == string(telemetryapi.Extension) {
```

Comment:
I thought we agreed not to forward Extension events as logs?

Reply:
Yes, the logic is the same. The decision now is made from the subscription type. 

https://github.com/solarwinds/opentelemetry-lambda/blob/no_platform/collector/receiver/telemetryapireceiver/receiver.go#L74

Reply:
I see. Can we just not subscribe to anything other than Function events for this round to minimize overhead?

Reply:
Sure. Done